packages/admin/src/DirtyHandlerApiContext.tsx

```diff
@@ -1,9 +1,10 @@
+import { Location } from "history";
 import * as React from "react";
 
 import { SubmitResult } from "./form/SubmitResult";
 
 export interface IDirtyHandlerApiBinding {
-    isDirty: () => boolean;
+    isDirty: (location?: Location) => boolean;
```

Comment:
I don't understand how this works :)

what location is passed here? can you name it more specific?

Reply:
You're right, it's not logical to pass location into isDirty. Have a look if you like it this way or whether we should type the payload for isDirty.